seedlet/style-rtl.css

```
@@ -2400,16 +2396,24 @@ table th,
  * .alignright
  */
 .alignright {
-	float: right;
 	margin-top: 0;
 	margin-bottom: var(--global--spacing-vertical);
-	margin-left: var(--global--spacing-horizontal);
 }
 
 .entry-content > .alignright {
 	max-width: var(--responsive--aligndefault-width);
 }
 
+@media only screen and (min-width: 482px) {
+	.alignright {
+		float: right;
+		margin-left: var(--global--spacing-horizontal);
+	}
+	.entry-content > .alignright {
+		max-width: calc(50% - var(--responsive--alignrightleft-width));
```

Do you think it's strange that we are doing another `calc` here? Could we just run one `calc` in the responsive styles, so that this line could look like this instead:

```suggestion
		max-width: var(--responsive--alignrightleft-width);
```

---

(Whoops, this should be a suggestion on the sass file)

---

If I do that I'd have to make a calculation here: https://github.com/Automattic/themes/blob/2a25779dd771b417d5e95a273bdddf9365ac9aaf/seedlet/assets/sass/structure/_responsive-logic.scss#L146

---

Hmm why not here? 

https://github.com/Automattic/themes/blob/2a25779dd771b417d5e95a273bdddf9365ac9aaf/seedlet/assets/sass/structure/_responsive-logic.scss#L94